src/person/controllers/update_person_controller.py

```diff
@@ -73,6 +73,8 @@ def process(self):
                 days = pause_weeks * 6
                 self._person.pause_expiration = datetime.now() + timedelta(days=days)
 
+        self._person.last_active = datetime.now()
```

Comment:
is this accounting for time zone differences? just want to make sure this is ET

Reply:
good point! you can get the timezone being used in Django from the settings file and use that for datetime